methods_and_flow_control/fizz_buzz.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+
+# Adding these methods to Integers makes the whole read easier.
+class Fixnum
+  def divisible_by?(divisor)
+    if self % divisor == 0
```

Comment:
`self % divisor == 0` is all you need for this method.  That line will return true/false for you.